cmd/client/main.go

```diff
@@ -96,6 +99,8 @@ func (o *GrpcProxyClientOptions) Flags() *pflag.FlagSet {
 	flags.StringVar(&o.userAgent, "user-agent", o.userAgent, "User agent to pass to the proxy server")
 	flags.IntVar(&o.testRequests, "test-requests", o.testRequests, "The number of times to send the request.")
 	flags.IntVar(&o.testDelaySec, "test-delay", o.testDelaySec, "The delay in seconds between sending requests.")
+	flags.IntVar(&o.afterDelaySec, "after-delay", o.afterDelaySec, "The delay in seconds after sending requests.")
+	flags.BoolVar(&o.closeIdleConn, "close-idle-conn", 	o.closeIdleConn, "Controls if the client calls closeIdleConnections.")
```

Comment:
There's a weird space here, is there an extra tab?

Reply:
Fixed.